test/CMakeLists.txt

```diff
@@ -1,35 +1,39 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Quick note on the version requirement bump:
This is effectively not-a-change because we include `google/googletest:master`, which itself requires `3.5` already.

Reply:
Good to know, my initial suspicion was that some fancier CMake feature was now being used, but since the version was old enough I didn't bother looking into it more.